nx_parallel/algorithms/centrality/harmonic.py

```diff
@@ -0,0 +1,64 @@
+from joblib import Parallel, delayed
+import networkx as nx
+import nx_parallel as nxp
+from functools import partial
+
+__all__ = ["harmonic_centrality"]
+
+
+@nxp._configure_if_nx_active()
+def harmonic_centrality(
+    G, nbunch=None, distance=None, sources=None, get_chunks="chunks"
+):
+    """The parallel computation is implemented by dividing the nodes into chunks and
+    computing harmonic centrality for each chunk concurrently.
+
+    networkx.harmonic_centrality : https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.centrality.harmonic_centrality.html
+
+    Parameters
+    ----------
+    get_chunks : str, function (default = "chunks")
+        A function that takes in a list of all the nodes as input and returns an
+        iterable `node_chunks`. The default chunking is done by slicing the
+        `nodes` into `n_jobs` number of chunks.
+    """
+
+    def _process_chunk(chunk):
+        hc = {}
+        for v in chunk:
+            dist = spl(v)
+            for u, d_uv in dist.items():
+                if d_uv != 0 and u in nbunch:
+                    node = v if transposed else u
+                    hc[node] = (hc[node] + 1 / d_uv) if node in hc else (1 / d_uv)
+        return hc
+
+    if hasattr(G, "graph_object"):
+        G = G.graph_object
+
+    nbunch = set(G.nbunch_iter(nbunch) if nbunch is not None else G.nodes)
+    sources = set(G.nbunch_iter(sources) if sources is not None else G.nodes)
```

Comment:
The logic for handling `nbunch is None` is already included in `G.nbunch_iter`. And we are creating a set out of the iterator anyway, so I think it's just as performant to use: `nbunch = set(G.nbunch_iter(nbunch))` and similarly for `sources`.